swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+
+from swap_meet.item import Item
+
+class Clothing(Item):
+
+    def __init__(self, id = None, condition = None, fabric = None):
+        super().__init__(id, condition)
+        self.fabric = "Unknown" if fabric == None else fabric
+
+    def __str__(self):
+        return f"An object of type Clothing with id {self.id}. It is made from {self.fabric} fabric."
```

Comment:
Notice that you have four places that have a string "An object of type ____ with id ____" and then you hardcoded the type (in Item, Electronics, Decor, and Clothing classes). 

What if your child classes called the `__str__()` method from its parent class and then added some additional functionality within the child class so you don't repeat the part "An object of type ____ with id ____."

That would be like:
```python
def __str__(self):
       object_type_message = super().__str__()
       clothing_message = f"An object of type Clothing with id {self.id}. It is made from {self.fabric} fabric."
       return object_type_message + " " + clothing_message
```

This does mean you need to refactor the __str__ method in your parent class to be more dynamic so look in that class for my other comment about how to refactor it